docs/providers.md

```diff
@@ -147,3 +147,37 @@ There are two required labels for OpenShift to run an Atomic App.
   * run the container to allow the image to determine what OpenShift objects are created.
 1. `io.openshift.generate.token.as=env:TOKEN_ENV_VAR`
   * run the container with user token so `oc` commands can be run on behalf of the user from within the container pod.
+
+
+
+### Marathon
+
+**Features**
+
+The Marathon provider will deploy an application into Mesos cluster using Marathon
+scheduler.
+
+This provider requires configuration (`providerurl`) to be able to connect to Marathon API. If no `providerurl` is specified it will use `http://localhost:8080` as
+Marathon API url.
+
+**Configuration values**
+
+Table 1. Marathon default configuration values
+
+Keyword    | Required | Description                                 | Default value
+-----------|----------|---------------------------------------------|--------------------------
+providerurl|   no     |  url for Marathon REST API   | http://localhost:8080
```

Comment:
Right, so maybe `providerapi`? `providerurl` doesn't make sense to me. Let's see what others think:-)